beacon-chain/blockchain/receive_block_test.go

```diff
@@ -0,0 +1,157 @@
+package blockchain
+
+import (
+	"context"
+	"testing"
+
+	testDB "github.com/prysmaticlabs/prysm/beacon-chain/db/testing"
+
```

Comment:
goimport

Reply:
this is valid for goimport, I'll move it to the second group though